Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45044&attempt_id=a46eb374-60af-43d0-b36a-3d3cd5c33da5)

Attempted: rivet, yoda

<details><summary>Partial log (click to expand)</summary><p>

```
  File "/private/tmp/nix-build-rivet-2.6.1.drv-0/Rivet-2.6.1/pyext/build/lib.macosx-10.10-x86_64-2.7/rivet/__init__.py", line 12, in <module>
    from rivet.core import *
ImportError: dlopen(/private/tmp/nix-build-rivet-2.6.1.drv-0/Rivet-2.6.1/pyext/build/lib.macosx-10.10-x86_64-2.7/rivet/core.so, 10): Library not loaded: /nix/store/f01wy7kh5d1qcra2nw9dmlxnv5s5acsf-rivet-2.6.1/lib/libRivet.dylib
  Referenced from: /private/tmp/nix-build-rivet-2.6.1.drv-0/Rivet-2.6.1/pyext/build/lib.macosx-10.10-x86_64-2.7/rivet/core.so
  Reason: image not found
make[1]: *** [Makefile:597: analyses.html] Error 1
make[1]: Leaving directory '/private/tmp/nix-build-rivet-2.6.1.drv-0/Rivet-2.6.1/doc'
make: *** [Makefile:556: all-recursive] Error 1
builder for '/nix/store/b3a13zjk20x2j7pym9gskdcdnndd1qwz-rivet-2.6.1.drv' failed with exit code 2
error: build of '/nix/store/b3a13zjk20x2j7pym9gskdcdnndd1qwz-rivet-2.6.1.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/physics/rivet/default.nix
+++ b/pkgs/development/libraries/physics/rivet/default.nix
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "rivet-${version}";
-  version = "2.6.0";
+  version = "2.6.1";
 
   src = fetchurl {
     url = "https://www.hepforge.org/archive/rivet/Rivet-${version}.tar.bz2";
-    sha256 = "007rwal8wx2k9gs0r6kym6ix0siz0x9l55q9myq41siirpf2jcpv";
+    sha256 = "0ybv5dl3blgk9p9ayhknirxb6504kpygnkmizd5dr9ippys1i3c8";
   };
 
   patches = [
@@ -28,6 +28,10 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [ fastjet ghostscript gsl yoda ];
 
   preConfigure = ''
+    substituteInPlace Makefile.in \
+      --replace "SUBDIRS = src pyext data include bin analyses doc test" "SUBDIRS = src pyext data include bin analyses test"
+    substituteInPlace analyses/Makefile.in \
+      --replace "!(tmp)" ""
     substituteInPlace bin/rivet-buildplugin.in \
       --replace '"which"' '"${which}/bin/which"' \
       --replace 'mycxx=' 'mycxx=${stdenv.cc}/bin/${if stdenv.cc.isClang or false then "clang++" else "g++"}  #' \

